scripts/env/pull-env.sh

```
@@ -46,30 +49,31 @@ shift $((OPTIND - 1))
 
-if [ ! -f ".env.local" ]; then
-    cat <<EOF > .env.local
+if [ ! -f "$ENV_LOCAL_FILE" ]; then
+    cat <<EOF > $ENV_LOCAL_FILE
 # Local environment variables
 # Get your infisical service token on your provided account
 INFISICAL_TOKEN=''
 EOF
-  err_token_missing
+  err_token_missing $ENV_LOCAL_FILE
   exit
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)